src/FreeScribe.client/UI/SettingsWindowUI.py

```diff
@@ -551,6 +552,10 @@ def save_settings(self, close_window=True):
             self.cutoff_slider.threshold / 32768,
         )
 
+        # if Local Whisper is selected, load the Whisper Model
+        if self.settings.editable_settings["Local Whisper"]:
+            self.root.event_generate("<<LoadSttModel>>")
```

Comment:
might also need to check if the model has changed. and load only if the model has changed